ast/internal/tokens/tokens.go

```diff
@@ -67,6 +67,8 @@ const (
 	Semicolon
 
 	Every
+	Contains
+	If
```

Comment:
This adds the new keywords to the keyword enum. They won't be parsed until they're activated, though, by the same means as `in` and `every`.